apps/google-analytics-4/frontend/src/components/main-app/ChartContent.tsx

```diff
@@ -15,9 +15,11 @@ const ChartContent = (props: Props) => {
   const parseRowDates = (): string[] => {
     return pageViewData.rows.map((r: Row) => {
       const d = r.dimensionValues[0].value;
-      return new Date(
+      const utcDate = new Date(
         `${d.substring(0, 4)}-${d.substring(4, 6)}-${d.substring(6, 8)}`
-      ).toLocaleDateString('en-us', { month: 'short', day: 'numeric' });
+      ).toUTCString();
```

Comment:
I know you have more context, but I'd seriously consider here and use toISOString() and parse that to get the UTC month and day you need.
https://stackoverflow.com/questions/58847869/utc-vs-iso-format-for-time

Reply:
Additionally if you do something like this 
```
date.toISOString().split("T")[0] // outputs something like '2023-03-16'
```

You get this and maybe you can write a function that returns a tuple of day and month based on this parsing of the ISO string since this substring logic is done twice